src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import static core.basesyntax.ColorSupplier.RANDOM;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MIN_LENGTH = 2;
+    private static final int MAX_LENGTH = 10;
+
+    public Figure getRandomFigure() {
+        int ran = RANDOM.nextInt(FIGURE_COUNT);
+        int a = RANDOM.nextInt(MAX_LENGTH - MIN_LENGTH + 1) + MIN_LENGTH;
+        int b = RANDOM.nextInt(MAX_LENGTH - MIN_LENGTH + 1) + MIN_LENGTH;
+        int c = RANDOM.nextInt(MAX_LENGTH - MIN_LENGTH + 1) + MIN_LENGTH;
+        switch (ran) {
+            case (0):
+                return new Square(a);
+            case (1):
+                return new Rectangle(a, b);
+            case (2):
+                return new RightTriangle(a, b);
+            case (3):
+                return new Circle(a, Color.WHITE);

```

Comment:
Why you allow to pass color in constructor for Circle but not the rest of figures?